recipes/arvados-python-client/meta.yaml

```diff
@@ -10,33 +10,29 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
 
 requirements:
   host:
     - python >=3
-    - setuptools
-    - ciso8601 >=2.0.0
-    - future
-    - google-api-python-client <1.7,>=1.6.2
-    - google-auth <2
-    - httplib2
-    - pycurl >=7.19.5.1,<7.45.0
-    - ruamel.yaml >=0.15.54
-    - ws4py >=0.4.2
-    - protobuf <4.0.0
+    - setuptools >=40.3.0
+    - pip
   run:
     - python >=3
-    - setuptools
+    - setuptools >=40.3.0
```

Comment:
Is `setuptools` needed as a run dependency at all?

Reply:
It is [listed](https://github.com/arvados/arvados/blob/main/sdk/python/setup.py#L131) in the `install_requires` section of the `setup.py` file.

I do not know if it is actually used though.

Either way, I am not sure it matters much given that `setuptools` is a dependency of the `python` conda package and is always dragged in.